RESTResource/RESTResource.js

```diff
@@ -734,6 +734,14 @@ export default class RESTResource {
 
       const { headers, records, resourceShouldRefresh } = options;
       const requestIndex = options.resultOffset >= 0 ? options.resultOffset : options.recordsRequired;
+
+      // If the offset in the current request matches the resultOffset
+      // of the previous request just skip it.
+      // https://issues.folio.org/browse/STCON-118
+      if (state[key].offset && state[key].offset === requestIndex) {
```

Comment:
Nice stop-gap solution! You might want to throw in some null safety ...just in case...

`state[key]?.offset` or similar

Reply:
good idea!